hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/resolver/MountTableResolver.java

```diff
@@ -441,9 +448,12 @@ public PathLocation getDestinationForPath(final String path)
       if (this.locationCache == null) {
         res = lookupLocation(processTrashPath(path));
       } else {
-        Callable<? extends PathLocation> meh = (Callable<PathLocation>) () ->
-            lookupLocation(processTrashPath(path));
+        Callable<? extends PathLocation> meh = (Callable<PathLocation>) () -> {
+          LOC_CACHE_MISS.increment();
+          return lookupLocation(processTrashPath(path));
+        };
         res = this.locationCache.get(processTrashPath(path), meh);
+        LOC_CACHE_ACCESS.increment();
```

Comment:
Can we add some tests?

Reply:
OK, i will add unit tests soon.

Reply:
hi, @goiri @tomscut , sorry for responding so late.  i have add an unit test to test the cache access.